client/my-sites/checkout/checkout-thank-you/hundred-year-plan-thank-you/index.tsx

```diff
@@ -195,13 +196,16 @@ export default function HundredYearPlanThankYou( { siteSlug, receiptId }: Props
 					<Content isMobile={ isMobile }>
 						<div className="hundred-year-plan-thank-you__thank-you-text-container">
 							<Header className="wp-brand-font" isMobile={ isMobile }>
-								{ translate( 'Your legacy is in safe hands' ) }
+								{ translate( 'Your century-long legacy begins now' ) }
 							</Header>
 							<Highlight isMobile={ isMobile }>
 								{ translate(
-									'Your 100-year legacy for %(domain)s begins now. Our dedicated premium support team will reach out via your email shortly to schedule a welcome session, and walk you through the benefits tailored just for you. We are thrilled to stand by your side for the next century.',
+									'The %(planTitle)s for (%(domain)s) is active. Our Premier Support team will be in touch by email shortly to schedule a welcome session and walk you through your exclusive benefits. We’re looking forward to supporting you every step of the way.',
 									{
-										args: { domain: registeredDomain?.domain || siteSlug },
+										args: {
+											domain: registeredDomain?.domain || siteSlug,
+											planTitle: getPlan( PLAN_100_YEARS )?.getTitle() || '',
```

Comment:
I think it is unnecessary to make this plan name dynamic. This thank you page is a special case laser focussed for the 100 year plan. If the need arises in the future we can switch back to making this dynamic. But right now I think it is unnecessary.

Reply:
I think including the hundred year name in the translation can even be beneficial so that its given an appropriate translated name as well.

Reply:
The plan name is already translated:
https://github.com/Automattic/wp-calypso/blob/ca088cf591add9173bd291c98e08dc8d220dc748/packages/calypso-products/src/plans-list.tsx#L2721

I'm using this function so that the same translated name is used everywhere.